### ISSUE TITLE
In order to save your changes, all s must be a .jpeg, .jpg, or .png file that is @@DIMENSION_VALUE@@ pixels, at least 72 DPI, and in the RGB color space. There is an error for 1 of your localizations.

### ISSUE THREAD

**Issue by combinatorial** [fastlane/fastlane#5355]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

There are a number of issues about this that are closed. Intermittently I get this error, re-running often gets it to go away. In order to successfully upload I have to run multiple times, and delete the metadata that succeeded each time.

##### Complete output when running fastlane, including the stack trace and command used

[10:04:56]: -------------------------------------------------
[10:04:56]: --- Step: Verifying required fastlane version ---
[10:04:56]: -------------------------------------------------
[10:04:56]: fastlane version valid
[10:04:56]: ------------------------------
[10:04:56]: --- Step: default_platform ---
[10:04:56]: ------------------------------
[10:04:57]: Driving the lane 'ios update_metadata' 🚀
[10:04:57]: ---------------------
[10:04:57]: --- Step: deliver ---
[10:04:57]: ---------------------
[10:04:57]: To not be asked about this value, you can specify it using 'username'
Your Apple ID Username: xxxx
[10:05:04]: Login to iTunes Connect (xxxx)
[10:05:06]: Login successful

+--------------------+------------------------+
|           deliver 1.13.1 Summary            |
+--------------------+------------------------+
| app_identifier     | xxxx |
| username           | xxxx     |
| screenshots_path   | ./fastlane/screenshots |
| metadata_path      | ./fastlane/metadata    |
| skip_binary_upload | false                  |
| skip_screenshots   | false                  |
| skip_metadata      | false                  |
| force              | false                  |
| submit_for_review  | false                  |
| automatic_release  | false                  |
| dev_portal_team_id | xxxx             |
+--------------------+------------------------+

[10:05:07]: Loading './fastlane/metadata/es-ES/description.txt'...
[10:05:07]: Loading './fastlane/metadata/es-ES/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/es-ES/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/es-ES/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/es-ES/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/es-ES/name.txt'...
[10:05:07]: Loading './fastlane/metadata/es-ES/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/es-MX/description.txt'...
[10:05:07]: Loading './fastlane/metadata/es-MX/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/es-MX/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/es-MX/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/es-MX/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/es-MX/name.txt'...
[10:05:07]: Loading './fastlane/metadata/es-MX/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/fi/description.txt'...
[10:05:07]: Loading './fastlane/metadata/fi/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/fi/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/fi/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/fi/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/fi/name.txt'...
[10:05:07]: Loading './fastlane/metadata/fi/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/fr-FR/description.txt'...
[10:05:07]: Loading './fastlane/metadata/fr-FR/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/fr-FR/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/fr-FR/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/fr-FR/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/fr-FR/name.txt'...
[10:05:07]: Loading './fastlane/metadata/fr-FR/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/it/description.txt'...
[10:05:07]: Loading './fastlane/metadata/it/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/it/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/it/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/it/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/it/name.txt'...
[10:05:07]: Loading './fastlane/metadata/it/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/ja/description.txt'...
[10:05:07]: Loading './fastlane/metadata/ja/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/ja/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/ja/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/ja/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/ja/name.txt'...
[10:05:07]: Loading './fastlane/metadata/ja/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/ko/description.txt'...
[10:05:07]: Loading './fastlane/metadata/ko/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/ko/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/ko/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/ko/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/ko/name.txt'...
[10:05:07]: Loading './fastlane/metadata/ko/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-BR/description.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-BR/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-BR/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-BR/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-BR/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-BR/name.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-BR/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-PT/description.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-PT/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-PT/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-PT/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-PT/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-PT/name.txt'...
[10:05:07]: Loading './fastlane/metadata/pt-PT/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/ru/description.txt'...
[10:05:07]: Loading './fastlane/metadata/ru/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/ru/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/ru/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/ru/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/ru/name.txt'...
[10:05:07]: Loading './fastlane/metadata/ru/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hans/description.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hans/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hans/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hans/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hans/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hans/name.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hans/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hant/description.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hant/keywords.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hant/release_notes.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hant/support_url.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hant/marketing_url.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hant/name.txt'...
[10:05:07]: Loading './fastlane/metadata/zh-Hant/privacy_url.txt'...
[10:05:07]: Loading './fastlane/metadata/copyright.txt'...
[10:05:07]: Loading './fastlane/metadata/primary_category.txt'...
[10:05:07]: Loading './fastlane/metadata/secondary_category.txt'...
[10:05:07]: Loading './fastlane/metadata/primary_first_sub_category.txt'...
[10:05:07]: Loading './fastlane/metadata/primary_second_sub_category.txt'...
[10:05:07]: Loading './fastlane/metadata/secondary_first_sub_category.txt'...
[10:05:07]: Loading './fastlane/metadata/secondary_second_sub_category.txt'...
[10:05:07]: Verifying the upload via the HTML file can be disabled by either adding
[10:05:07]: `force true` to your Deliverfile or using `deliver --force`
Does the Preview on path './Preview.html' look okay for you? (y/n)
y
 [10:05:14]: HTML file confirmed...
[10:05:16]: Activating languages es-ES, es-MX, fi, fr-FR, it, ja, ko, pt-BR, pt-PT, ru, zh-Hans, zh-Hant...
[10:05:21]: Uploading metadata to iTunes Connect
[10:05:24]: Successfully uploaded initial set of metadata to iTunes Connect
[10:05:27]: Starting with the upload of screenshots...
[10:05:27]: Uploading 30 screenshots for language es-ES
[10:05:27]: Uploading './fastlane/screenshots/es-ES/1_iPad.png'...
[10:05:31]: Uploading './fastlane/screenshots/es-ES/1_iPadPro.png'...
[10:05:40]: Uploading './fastlane/screenshots/es-ES/2_iPad.png'...
[10:05:43]: Uploading './fastlane/screenshots/es-ES/2_iPadPro.png'...
[10:05:49]: Uploading './fastlane/screenshots/es-ES/3_iPad.png'...
[10:05:52]: Uploading './fastlane/screenshots/es-ES/3_iPadPro.png'...
[10:05:59]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone4S-es-1.png'...
[10:06:00]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone4S-es-2.png'...
[10:06:02]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone4S-es-3.png'...
[10:06:03]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone4S-es-4.png'...
[10:06:04]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone4S-es-5.png'...
[10:06:05]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone5S-es-1.png'...
[10:06:06]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone5S-es-2.png'...
[10:06:07]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone5S-es-3.png'...
[10:06:08]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone5S-es-4.png'...
[10:06:11]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone5S-es-5.png'...
[10:06:12]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone6S-es-1.png'...
[10:06:15]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone6S-es-2.png'...
[10:06:17]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone6S-es-3.png'...
[10:06:19]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone6S-es-4.png'...
[10:06:20]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone6S-es-5.png'...
[10:06:22]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone6SPlus-es-1.png'...
[10:06:30]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone6SPlus-es-2.png'...
[10:06:33]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone6SPlus-es-3.png'...
[10:06:36]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone6SPlus-es-4.png'...
[10:06:39]: Uploading './fastlane/screenshots/es-ES/4.1-iPhone6SPlus-es-5.png'...
[10:06:41]: Uploading './fastlane/screenshots/es-ES/4_iPad.png'...
[10:06:44]: Uploading './fastlane/screenshots/es-ES/4_iPadPro.png'...
[10:06:49]: Uploading './fastlane/screenshots/es-ES/5_iPad.png'...
[10:06:53]: Uploading './fastlane/screenshots/es-ES/5_iPadPro.png'...
[10:06:59]: Saving changes
[10:08:12]: Uploading 30 screenshots for language es-MX
[10:08:12]: Uploading './fastlane/screenshots/es-MX/1_iPad.png'...
[10:08:15]: Uploading './fastlane/screenshots/es-MX/1_iPadPro.png'...
[10:08:20]: Uploading './fastlane/screenshots/es-MX/2_iPad.png'...
[10:08:24]: Uploading './fastlane/screenshots/es-MX/2_iPadPro.png'...
[10:08:29]: Uploading './fastlane/screenshots/es-MX/3_iPad.png'...
[10:08:33]: Uploading './fastlane/screenshots/es-MX/3_iPadPro.png'...
[10:08:39]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone4S-es-1.png'...
[10:08:40]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone4S-es-2.png'...
[10:08:41]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone4S-es-3.png'...
[10:08:42]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone4S-es-4.png'...
[10:08:43]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone4S-es-5.png'...
[10:08:47]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone5S-es-1.png'...
[10:08:48]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone5S-es-2.png'...
[10:08:50]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone5S-es-3.png'...
[10:08:52]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone5S-es-4.png'...
[10:08:53]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone5S-es-5.png'...
[10:08:54]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone6S-es-1.png'...
[10:08:59]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone6S-es-2.png'...
[10:09:01]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone6S-es-3.png'...
[10:09:06]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone6S-es-4.png'...
[10:09:08]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone6S-es-5.png'...
[10:09:09]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone6SPlus-es-1.png'...
[10:09:12]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone6SPlus-es-2.png'...
[10:09:15]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone6SPlus-es-3.png'...
[10:09:19]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone6SPlus-es-4.png'...
[10:09:23]: Uploading './fastlane/screenshots/es-MX/4.1-iPhone6SPlus-es-5.png'...
[10:09:29]: Uploading './fastlane/screenshots/es-MX/4_iPad.png'...
[10:09:32]: Uploading './fastlane/screenshots/es-MX/4_iPadPro.png'...
[10:09:39]: Uploading './fastlane/screenshots/es-MX/5_iPad.png'...
[10:09:43]: Uploading './fastlane/screenshots/es-MX/5_iPadPro.png'...
[10:09:49]: Saving changes
[10:10:47]: Uploading 30 screenshots for language fi
[10:10:47]: Uploading './fastlane/screenshots/fi/1_iPad.png'...
[10:10:50]: Uploading './fastlane/screenshots/fi/1_iPadPro.png'...
[10:10:55]: Uploading './fastlane/screenshots/fi/2_iPad.png'...
[10:11:02]: Uploading './fastlane/screenshots/fi/2_iPadPro.png'...
[10:11:07]: Uploading './fastlane/screenshots/fi/3_iPad.png'...
[10:11:11]: Uploading './fastlane/screenshots/fi/3_iPadPro.png'...
[10:11:17]: Uploading './fastlane/screenshots/fi/4.1-iPhone4S-fi-1.png'...
[10:11:18]: Uploading './fastlane/screenshots/fi/4.1-iPhone4S-fi-2.png'...
[10:11:18]: Uploading './fastlane/screenshots/fi/4.1-iPhone4S-fi-3.png'...
[10:11:19]: Uploading './fastlane/screenshots/fi/4.1-iPhone4S-fi-4.png'...
[10:11:20]: Uploading './fastlane/screenshots/fi/4.1-iPhone4S-fi-5.png'...
[10:11:21]: Uploading './fastlane/screenshots/fi/4.1-iPhone5S-fi-1.png'...
[10:11:22]: Uploading './fastlane/screenshots/fi/4.1-iPhone5S-fi-2.png'...
[10:11:28]: Uploading './fastlane/screenshots/fi/4.1-iPhone5S-fi-3.png'...
[10:11:29]: Uploading './fastlane/screenshots/fi/4.1-iPhone5S-fi-4.png'...
[10:11:30]: Uploading './fastlane/screenshots/fi/4.1-iPhone5S-fi-5.png'...
[10:11:31]: Uploading './fastlane/screenshots/fi/4.1-iPhone6S-fi-1.png'...
[10:11:33]: Uploading './fastlane/screenshots/fi/4.1-iPhone6S-fi-2.png'...
[10:11:34]: Uploading './fastlane/screenshots/fi/4.1-iPhone6S-fi-3.png'...
[10:11:40]: Uploading './fastlane/screenshots/fi/4.1-iPhone6S-fi-4.png'...
[10:11:42]: Uploading './fastlane/screenshots/fi/4.1-iPhone6S-fi-5.png'...
[10:11:43]: Uploading './fastlane/screenshots/fi/4.1-iPhone6SPlus-fi-1.png'...
[10:11:48]: Uploading './fastlane/screenshots/fi/4.1-iPhone6SPlus-fi-2.png'...
[10:11:51]: Uploading './fastlane/screenshots/fi/4.1-iPhone6SPlus-fi-3.png'...
[10:11:54]: Uploading './fastlane/screenshots/fi/4.1-iPhone6SPlus-fi-4.png'...
[10:12:02]: Uploading './fastlane/screenshots/fi/4.1-iPhone6SPlus-fi-5.png'...
[10:12:04]: Uploading './fastlane/screenshots/fi/4_iPad.png'...
[10:12:09]: Uploading './fastlane/screenshots/fi/4_iPadPro.png'...
[10:12:13]: Uploading './fastlane/screenshots/fi/5_iPad.png'...
[10:12:16]: Uploading './fastlane/screenshots/fi/5_iPadPro.png'...
[10:12:23]: Saving changes
[10:13:25]: Uploading 30 screenshots for language fr-FR
[10:13:25]: Uploading './fastlane/screenshots/fr-FR/1_iPad.png'...
[10:13:28]: Uploading './fastlane/screenshots/fr-FR/1_iPadPro.png'...
[10:13:34]: Uploading './fastlane/screenshots/fr-FR/2_iPad.png'...
[10:13:37]: Uploading './fastlane/screenshots/fr-FR/2_iPadPro.png'...
[10:13:42]: Uploading './fastlane/screenshots/fr-FR/3_iPad.png'...
[10:13:46]: Uploading './fastlane/screenshots/fr-FR/3_iPadPro.png'...
[10:13:54]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone4S-fr-1.png'...
[10:13:55]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone4S-fr-2.png'...
[10:13:56]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone4S-fr-3.png'...
[10:13:57]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone4S-fr-4.png'...
[10:13:58]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone4S-fr-5.png'...
[10:14:00]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone5S-fr-1.png'...
[10:14:01]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone5S-fr-2.png'...
[10:14:02]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone5S-fr-3.png'...
[10:14:04]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone5S-fr-4.png'...
[10:14:06]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone5S-fr-5.png'...
[10:14:07]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone6S-fr-1.png'...
[10:14:08]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone6S-fr-2.png'...
[10:14:10]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone6S-fr-3.png'...
[10:14:11]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone6S-fr-4.png'...
[10:14:14]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone6S-fr-5.png'...
[10:14:26]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone6SPlus-fr-1.png'...
[10:14:28]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone6SPlus-fr-2.png'...
[10:14:32]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone6SPlus-fr-3.png'...
[10:14:35]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone6SPlus-fr-4.png'...
[10:14:38]: Uploading './fastlane/screenshots/fr-FR/4.1-iPhone6SPlus-fr-5.png'...
[10:14:41]: Uploading './fastlane/screenshots/fr-FR/4_iPad.png'...
[10:14:44]: Uploading './fastlane/screenshots/fr-FR/4_iPadPro.png'...
[10:14:52]: Uploading './fastlane/screenshots/fr-FR/5_iPad.png'...
[10:14:57]: Uploading './fastlane/screenshots/fr-FR/5_iPadPro.png'...
[10:15:02]: Saving changes
[10:16:05]: Uploading 30 screenshots for language it
[10:16:05]: Uploading './fastlane/screenshots/it/1_iPad.png'...
[10:16:08]: Uploading './fastlane/screenshots/it/1_iPadPro.png'...
[10:16:14]: Uploading './fastlane/screenshots/it/2_iPad.png'...
[10:16:18]: Uploading './fastlane/screenshots/it/2_iPadPro.png'...
[10:16:23]: Uploading './fastlane/screenshots/it/3_iPad.png'...
[10:16:27]: Uploading './fastlane/screenshots/it/3_iPadPro.png'...
[10:16:34]: Uploading './fastlane/screenshots/it/4.1-iPhone4S-it-1.png'...
[10:16:34]: Uploading './fastlane/screenshots/it/4.1-iPhone4S-it-2.png'...
[10:16:35]: Uploading './fastlane/screenshots/it/4.1-iPhone4S-it-3.png'...
[10:16:36]: Uploading './fastlane/screenshots/it/4.1-iPhone4S-it-4.png'...
[10:16:37]: Uploading './fastlane/screenshots/it/4.1-iPhone4S-it-5.png'...
[10:16:38]: Uploading './fastlane/screenshots/it/4.1-iPhone5S-it-1.png'...
[10:16:39]: Uploading './fastlane/screenshots/it/4.1-iPhone5S-it-2.png'...
[10:16:40]: Uploading './fastlane/screenshots/it/4.1-iPhone5S-it-3.png'...
[10:16:42]: Uploading './fastlane/screenshots/it/4.1-iPhone5S-it-4.png'...
[10:16:43]: Uploading './fastlane/screenshots/it/4.1-iPhone5S-it-5.png'...
[10:16:44]: Uploading './fastlane/screenshots/it/4.1-iPhone6S-it-1.png'...
[10:16:45]: Uploading './fastlane/screenshots/it/4.1-iPhone6S-it-2.png'...
[10:16:47]: Uploading './fastlane/screenshots/it/4.1-iPhone6S-it-3.png'...
[10:16:50]: Uploading './fastlane/screenshots/it/4.1-iPhone6S-it-4.png'...
[10:16:51]: Uploading './fastlane/screenshots/it/4.1-iPhone6S-it-5.png'...
[10:16:53]: Uploading './fastlane/screenshots/it/4.1-iPhone6SPlus-it-1.png'...
[10:16:56]: Uploading './fastlane/screenshots/it/4.1-iPhone6SPlus-it-2.png'...
[10:16:59]: Uploading './fastlane/screenshots/it/4.1-iPhone6SPlus-it-3.png'...
[10:17:01]: Uploading './fastlane/screenshots/it/4.1-iPhone6SPlus-it-4.png'...
[10:17:03]: Uploading './fastlane/screenshots/it/4.1-iPhone6SPlus-it-5.png'...
[10:17:09]: Uploading './fastlane/screenshots/it/4_iPad.png'...
[10:17:12]: Uploading './fastlane/screenshots/it/4_iPadPro.png'...
[10:17:16]: Uploading './fastlane/screenshots/it/5_iPad.png'...
[10:17:20]: Uploading './fastlane/screenshots/it/5_iPadPro.png'...
[10:17:26]: Saving changes
[10:17:27]: Variable Dump:
[10:17:27]: {:DEFAULT_PLATFORM=>:ios, :PLATFORM_NAME=>:ios, :LANE_NAME=>"ios update_metadata"}
[10:17:27]: In order to save your changes, all s must be a .jpeg, .jpg, or .png file that is @@DIMENSION_VALUE@@ pixels, at least 72 DPI, and in the RGB color space. There is an error for 1 of your localizations.

+------+-------------------------------------+-------------+
|                     fastlane summary                     |
+------+-------------------------------------+-------------+
| Step | Action                              | Time (in s) |
+------+-------------------------------------+-------------+
| 1    | Verifying required fastlane version | 0           |
| 2    | default_platform                    | 0           |
| 3    | deliver                             | 749         |
+------+-------------------------------------+-------------+

[10:17:27]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

➡️  In order to save your changes, all s must be a .jpeg, .jpg, or .png file that is @@DIMENSION_VALUE@@ pixels , at least 72 DPI, and in the RGB color space - deliver 1.1.0
   https://github.com/fastlane/fastlane/issues/1590 [closed] 42 💬
   3 weeks ago

➡️  Documentation - ipa vs. ipa (shenzhen)?
   https://github.com/fastlane/fastlane/issues/72 [closed] 2 💬
   23 Feb 2015

➡️  Improve Beta Setup
   https://github.com/fastlane/fastlane/pull/1 [closed] 1 💬
   16 Dec 2014

[!] The request could not be completed because:
	In order to save your changes, all s must be a .jpeg, .jpg, or .png file that is @@DIMENSION_VALUE@@ pixels, at least 72 DPI, and in the RGB color space. There is an error for 1 of your localizations.
```
INSERT OUTPUT HERE
```

### Configuration Files

Please copy the complete content of your `Fastfile` and any other configuration files you use below:

Fastfile:
```ruby
# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "1.55.0"

default_platform :ios

platform :ios do
  before_all do
    # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
    #cocoapods
  end

  desc "Send screenshots and metadata to the iTunes Connect"
  lane :update_metadata do
    # snapshot
    deliver
  end

  # You can define as many lanes as you want

  after_all do |lane|
    # This block is called, only if the executed lane was successful

    # slack(
    #   message: "Successfully deployed new App Update."
    # )
  end

  error do |lane, exception|
    # slack(
    #   message: exception.message,
    #   success: false
    # )
  end
end



# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/docs/Platforms.md
# All available actions: https://github.com/fastlane/fastlane/blob/master/docs/Actions.md```

### Environment

fastlane version (run `fastlane -v`): fastlane 1.97.2

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)? no

Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)? no

**Comment by oyvkva** [comment]
I get the same error, happens after 4-6 languages or so, seems random.

**Comment by none2011** [comment]
+1
and it happens very often, even when I had very good network connection quality.

**Comment by lukas2** [comment]
+1

**Comment by TKBurner** [comment]
Hi @combinatorial! Great to hear from you. I wanted to see if you can supply a project this is reproducible for. That way we can test over here. Thanks so much! 👍

**Comment by combinatorial** [comment]
Is there a way I can share with you privately?

**Comment by TKBurner** [comment]
@combinatorial Feel free to email me at support@fabric.io. Just mention it is for Todd and link this ticket. Thanks!

**Comment by cptdanko** [comment]
I have had this problem before and the way I have solved it was by moving the screenshots already uploaded from the screenshots directory to another one and re-running the process(lane). So the next time it resumes the upload process from where it stopped the last time and for me it worked the second time. I was working on a deadline and that "hack" did solve my problem and saved me some time.

**Comment by TKBurner** [comment]
@combinatorial Can you try @cptdanko workaround? Thanks!

**Comment by combinatorial** [comment]
You mean the workaround that was in the original report :) "In order to successfully upload I have to run multiple times, and delete the metadata that succeeded each time."

**Comment by TKBurner** [comment]
@combinatorial Thanks for following up. Were you able to put together an affected sample project? Thanks! 👍

**Comment by combinatorial** [comment]
Yes I was. I sent the email yesterday. It referenced your name and this issue number in the subject. "For Todd: Issue 5355"

**Comment by TKBurner** [comment]
@combinatorial Awesome! I've got it, can you let me know exact repro steps. I want to loop in the core team. Thanks so much! 👍

**Comment by combinatorial** [comment]
Run `fastlane update_metadata` sometimes it works and sometimes it fails with the error in the report. It's random, but in a 13 language upload it failed about 6 times for me.

**Comment by TKBurner** [comment]
Thanks! I'll update you soon! 👍

**Comment by cptdanko** [comment]
Yes my bad, I may have missed that bit of info from the question anyway I don't think I am only one who missed that bit of info. If there is a solution to your problem, it would be good if you can share with everyone else here.

**Comment by TKBurner** [comment]
@combinatorial Just followed up with you via email to get some more info. Thanks!

@cptdanko If you have a sample project this is reproducible in that'll help a lot!

**Comment by oyvkva** [comment]
I have submitted 5 projects over the last 2 weeks, all have gotten the same error. I can upload a few languages, then I have to move them over to a different folder and "deliver" again so it continues. 

Some for iPad only, some for all devices. If you want more projects to look at I can send over a few as well, or anything else I can try to help fixing the issue.

**Comment by ardee19** [comment]
@combinatorial or for others who ended up in here, like myself, and might have the same issue as mine, make sure your app icon (this is the case for me) or maybe your screenshots doesn't have any transparencies and/or alphas. PNGs retain them and iTunes Connect does not allow it.

I got this error because my app icon has some transparencies and/or alphas. I had to convert them to .JPG (this removes the transparencies/alphas), then convert it again to .PNG since Fastlane only accepts .PNG files even though iTunes Connect accepts JPEG files. I tried opening a ticket for it but was closed, was told that it is Apple's policy to have them in PNG :)
